bigchaindb/pipelines/block.py

```diff
@@ -25,11 +56,35 @@ def delete_tx(self, tx):
         return tx
 
     def validate_tx(self, tx):
+        """Validate a transaction.
+
+        Args:
+            tx (dict): the transaction to validate.
+
+        Returns:
+            The transaction if valid, ``None`` otherwise.
+        """
         tx = self.bigchain.is_valid_transaction(tx)
         if tx:
             return tx
 
     def create(self, tx, timeout=False):
+        """Create a block.
+
+        This method accumulates transactions to put in a block and outputs
+        a block when one of the following conditions is true:
+        - the size limit of the block has been reached, or
+        - a timeout happened.
+
+        Args:
+            tx (dict): the transaction to validate, might be None if
+                a timeout happens.
+            timeout (bool): ``True`` if a timeout happened
+                (Default: ``False``).
+
+        Returns:
+            The block.
```

Comment:
It returns None if `self.txs` is an empty list, or (`self.txs` doesn't have 1000 elements and no timeout), I think.